tests/test_axisgrid.py

```diff
@@ -696,6 +696,19 @@ def f(grid, color):
         assert res == color
         assert g.figure.get_facecolor() == color
 
+    def test_tick_params(self):
+
+        g = ag.FacetGrid(self.df, row="a", col="b")
+        color = 'blue'
+        width = 2
+        g.tick_params(width=width, color=color)
+        for ax in g.axes.ravel():
```

Comment:
Super nit (more of an FYI): using `array.flat` is more idiomatic than `array.ravel()` for iteration.